### PR TITLE
Add documentation for diacritics setting. (#223)

### LINKED ISSUE #233: Document the diacritics feature

### PR DESCRIPTION
**Pull request by melaniekung**

### CODE REVIEW

File: user-manual/administer/settings.rst [hunk 1]
```diff
@@ -1404,6 +1405,68 @@ viewed in the templates that have been selected.
...
 :ref:`Back to top <settings>`
 
+.. _diacritics:
+
+Diacritics
+=================
+
+Diacritics, also known as accents, are dots and squiggles written above or
+below certain letters of the alphabet of many languages, indicating something
+about thir pronounciation. Languages using diacritics varies from using a single
```
Comment (fiver-watson) [inline user-manual/administer/settings.rst]: > Languages using diacritics varies

I think it should be "vary" here.

File: user-manual/administer/settings.rst [hunk 2]
```diff
@@ -1404,6 +1405,68 @@ viewed in the templates that have been selected.
...
+diacritic to multiple diacritics on a single letter.
+
+AtoM uses `Elasticsearch <https://www.elastic.co/products/elasticsearch>`__ (ES)
+as its search index which offers a variety of filters and analyzers to convert or
```
Comment (fiver-watson) [inline user-manual/administer/settings.rst]: optional nitpick: usually a "which" is preceded by a comma

File: user-manual/administer/settings.rst [hunk 3]
```diff
@@ -1404,6 +1405,68 @@ viewed in the templates that have been selected.
...
+strip strings into their latin equivalents for searching purposes. By default,
```
Comment (fiver-watson) [inline user-manual/administer/settings.rst]: should Latin be capitalized here?

File: user-manual/administer/settings.rst [hunk 4]
```diff
@@ -1404,6 +1405,68 @@ viewed in the templates that have been selected.
...
+     - Ā => A
+     - Ą => A
+     - Ǎ => A
+     - Ç => C
+
+Note that YAML files are case sensitive and structured as a map or a list,
+following a hierarchy based on the indentation. The first line is an optional
+comment but the following 2 lines are required for AtoM to configure these mappings.
+The rest are examples of mapping special characters (on the left of the ``=>``)
+to its latin equivalent (on the right of the ``=>``) for ES to process. Each
```
Comment (fiver-watson) [inline user-manual/administer/settings.rst]: Same, re: Latin

**Review by fiver-watson — APPROVED** [review]
I noted some very minor suggestions grammar etc fixes, but they are not essential. Otherwise, LGTM!

File: user-manual/administer/settings.rst [hunk 5]
```diff
@@ -1404,6 +1405,68 @@ viewed in the templates that have been selected.
 
 :ref:`Back to top <settings>`
 
+.. _diacritics:
+
+Diacritics
+=================
+
+Diacritics, also known as accents, are dots and squiggles written above or
```
Comment (anvit) [inline user-manual/administer/settings.rst]: Minor nitpick, I'd say glyphs instead of squiggles

**Review by anvit — APPROVED** [review]
Had a minor nitpick, but looks good otherwise!

**Pull request merged**